tmpl/conversions_windows.tmpl

```diff
@@ -0,0 +1,110 @@
+//glow:keepspace
```

Comment:
It's not clear to me why this conversion code, or the debug code, needs to be forked across Windows/not-Windows.

Reply:
https://github.com/go-gl/glow/pull/102#discussion_r240010522 is the reason: using this `conversions_windows.tmpl` version at Cgo caused errors.

Reply:
Is there any way we can reduce the duplication by factoring out the parts that are the same? (Or am I misreading and all the implementations are subtly different?)

Reply:
Done (created the common `conversion.tmpl`)

Reply:
Getting back to this now (finally, sorry for the delay)...
1. Are the changes to conversions.tmpl platform agnostic and should be used everywhere?
2. I still don't quite understand why debug needs to be forked.

Reply:
Quick reply: https://github.com/go-gl/gl/issues/80 might be related, but I am not sure. I'll take a look later.

Reply:
I feel like if we can make conversions pure Go first, we should do this before merging this PR. Wdyt?

Reply:
Yeah, I'm all for updating conversions to work across the board.